src/routes/StackView/StackView.scss

```diff
@@ -53,6 +75,7 @@ $stack-view__parent-note-height: 22%;
 .stack-view__parent-note {
   margin: 0;
   height: $stack-view__parent-note-height;
+  min-height: 176px;
```

Comment:
oddly specific pixel height, I wonder what it is 🪄 🌈

Reply:
because it looks best lol

Reply:
Lol would you mind putting it into a variable? :)